File: python/MASTER_SLAVE_WHOLE_PART_PROXY/MASTER_SLAVE_WHOLE_PART_PROXY/whole_part/DatosCliente.py
class DatosCliente:
    
    nombres         : str
    apellidoPaterno : str
    apellidoMaterno : str
    edad            : int
    tipoCliente     : str
    
    def setDatosCliente(self, nombres         : str,
                              apellidoPaterno : str,
                              apellidoMaterno : str,
                              edad            : int,
                              tipoCliente      : bool) -> None:
        self.nombres         = nombres
        self.apellidoPaterno = apellidoPaterno
        self.apellidoMaterno = apellidoMaterno
        self.edad            = edad
        self.tipoCliente     = tipoCliente

File: php/BRIDGE_WHOLE_PART/whole_part/DatosCliente.php
<?php

class DatosCliente {

    public $nombre;
    public $apellidoPaterno;
    public $apellidoMaterno;
    public $edad;
    public $tipoCliente;
    
    public function setDatosCliente($nombre, $apellidoPaterno, $apellidoMaterno, $edad, $tipoCliente) {
        $this->nombre = $nombre;
        $this->apellidoPaterno = $apellidoPaterno;
        $this->apellidoMaterno = $apellidoMaterno;
        $this->edad = $edad;
        $this->tipoCliente = $tipoCliente;
    }
    public function getNombre() {
        return $this->nombre;
    }

    public function getApellidoPaterno() {
        return $this->apellidoPaterno;
    }

    public function getApellidoMaterno() {
        return $this->apellidoMaterno;
    }

    public function getEdad() {
        return $this->edad;
    }

    public function getTipoCliente() {
        return $this->tipoCliente;
    }


    

}
